apis/config.go

```diff
@@ -301,10 +304,37 @@ func DefaultAdvertiseClientURLs(cfg *EtcdAdmConfig) error {
 }
 
 // Returns the address associated with the host's default interface.
```

Comment:
I think the logic in the function is sound, but this function no longer does what its comment says it does.

```suggestion
// Returns the user-defined address; if that is undefined, returns the address associated with the host's default interface.
```

(I think refactoring this code would be nice, but not required for this PR.)

Reply:
pull request updated